R/qmd-utils.R

```diff
@@ -133,8 +133,10 @@ create_html_list <- function(string_to_append, list_values){
 #' 
 #' @return string with sets or parts depending on suffix of input_ID
 determine_link_source <- function(input_ID){
-  link_to_parts <- dictionary$parts_qmd_file_name
-  link_to_sets <- dictionary$sets_qmd_file_name
+  odm_dictionary$parts_qmd_file_name <- "parts"
```

Comment:
Why are you add these to the `odm_dictionary` object? Why can't you store them as independent variables?